iOS_4x_release.md

```diff
@@ -0,0 +1,48 @@
+# Aligning major version releases for the Adobe iOS SDK
+
+### Why are we getting another major update already?
+
+Apple has started doing an annual requirement to increase the minimum version of Xcode used when [submitting to the App Store](https://developer.apple.com/ios/submit/). Each Xcode version only supports back to a specific minimum version of iOS. Loosely speaking, the Xcode version and _minimum_ iOS version are correlated. 
+
+In order to align with Apple's changing requirements, we will also be doing an annual major release of the Adobe Experience Platform Mobile SDKs for iOS. In this yearly major update, you can expect the following changes:
+ - Increasing the minimum supported version of iOS.
+ - Increasing the minimum supported version of Xcode.
+ - Possible increase to minimum supported version of Swift.
```

Comment:
It feels a little awkward to have the possible here. Let's change this to "If necessary, increasing the minimum supported version of Swift."